server/tidb_library_test.go

```diff
@@ -49,7 +49,7 @@ func TestMemoryLeak(t *testing.T) {
 	runtime.ReadMemStats(&memStat)
 	// before the fix, initAndCloseTiDB for 20 times will cost 900 MB memory, so we test for a quite loose upper bound.
 	if syncutil.EnableDeadlock {
-		require.Less(t, memStat.HeapInuse-oldHeapInUse, uint64(1100*units.MiB))
+		require.Less(t, memStat.HeapInuse-oldHeapInUse, uint64(1500*units.MiB))
```

Comment:
Tests failed several times. Discussed with @hawkingrei to increase this value now and find a better solution later. 
```
Error:              "1158316032" is not less than "1153433600"
Error:              "1162543104" is not less than "1153433600"
```